pkg/util/fs_test.go

```diff
@@ -0,0 +1,111 @@
+package util
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// TestFileExists tests the FileExists utility function
+func TestFileExists(t *testing.T) {
+	tempDir := t.TempDir()
+
+	// Test 1: File exists
+	testFile := "test-file.txt"
+	os.WriteFile(filepath.Join(tempDir, testFile), []byte("test content"), 0644)
+
+	if !FileExists(tempDir, testFile) {
+		t.Errorf("Expected FileExists to return true for existing file")
+	}
+
+	// Test 2: File does not exist
+	nonExistentFile := "non-existent.txt"
```

Comment:
I don't think we need to be testing a simple boolean wrapper around a stdlib function.